src/main/java/com/yahoo/identity/services/storage/sql/SessionModel.java

```diff
@@ -0,0 +1,29 @@
+package com.yahoo.identity.services.storage.sql;
+
+import javax.annotation.Nonnull;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+public class SessionModel {
```

Comment:
SessionModel should just be a simple POJO. It's just used for working with MyBatis

Reply:
Sorry for the confusion caused by class name(there is actually no interaction with DB for session and credential exception `PasswordAbuseDection`). Now all the session and credential implementations have been moved to its own folder.